sql/moz-fx-data-shared-prod/apple_ads_external/ios_app_campaign_stats_v1/query.sql

```diff
@@ -2,61 +2,63 @@ WITH daily_stats AS (
   SELECT
     date_day AS `date`,
     campaign_name AS campaign,
-    campaign_id AS campaign_id,
+    campaign_id,
     REGEXP_EXTRACT(
       campaign_name,
       r'Mozilla_(?:FF|Firefox)_ASA_(?:iOSGeoTest_)?([\w]{2,3})_.*'
     ) AS campaign_country_code,
     ad_group_name,
     ad_group_id,
-    DATE_TRUNC(date_day, WEEK) AS week_start,
-    FORMAT_DATE("%b", date_day) AS `month`,
+    -- DATE_TRUNC(date_day, WEEK) AS week_start, -- TODO: this doesn't appear to be used later on, should it be?
+    -- FORMAT_DATE("%b", date_day) AS `month`, -- TODO: this doesn't appear to be used later on, should it be?
     SUM(spend) AS spend,
     SUM(taps) AS clicks,
     SUM(impressions) AS impressions,
-    SUM(redownloads) AS redownloads,
-    SUM(total_downloads) AS total_downloads
+    SUM(redownloads) AS redownloads, -- TODO: this doesn't appear to be used later on, is this needed?
+    SUM(total_downloads) AS total_downloads,
   FROM
     `moz-fx-data-shared-prod.apple_ads.ad_group_report`
+  WHERE
+    date_day = @submission_date
   GROUP BY
-    date_day,
+    `date`,
     campaign_name,
     campaign_id,
-    REGEXP_EXTRACT(campaign_name, r'Mozilla_(?:FF|Firefox)_ASA_(?:iOSGeoTest_)?([\w]{2,3})_.*'),
+    campaign_country_code,
     ad_group_name,
-    ad_group_id,
-    week_start,
-    `month`
+    ad_group_id
+    -- week_start,
+    -- `month`
 ),
 activations AS (
   SELECT
-    CAST(REGEXP_EXTRACT(cl.adjust_campaign, r' \((\d+)\)$') AS INT64) AS campaign_id,
+    clients.first_seen_date AS `date`,
+    CAST(REGEXP_EXTRACT(clients.adjust_campaign, r' \((\d+)\)$') AS INT64) AS campaign_id,
     CAST(REGEXP_EXTRACT(adjust_ad_group, r' \((\d+)\)$') AS INT64) AS ad_group_id,
-    cl.first_seen_date AS date,
     COUNT(DISTINCT ltv.client_id) AS new_profiles,
-    COUNTIF(cl.is_activated) AS activated,
-    SUM(ltv.lifetime_value) AS lifetime_value
+    COUNTIF(clients.is_activated) AS activated,
+    SUM(ltv.lifetime_value) AS lifetime_value,
   FROM
-    `mozdata.ltv.firefox_ios_client_ltv` ltv
+    `mozdata.ltv.firefox_ios_client_ltv` AS ltv
   INNER JOIN
-    `moz-fx-data-shared-prod.firefox_ios.firefox_ios_clients` cl
-    ON ltv.client_id = cl.client_id
-    AND ltv.sample_id = cl.sample_id
+    `moz-fx-data-shared-prod.firefox_ios.firefox_ios_clients` AS clients
+    USING(client_id, sample_id)
   WHERE
-    cl.channel = "release"
+    clients.channel = "release"
   GROUP BY
-    1,
-    2,
-    3
+    `date`,
+    campaign_id,
+    ad_group_id
 ),
 retention_aggs AS (
   SELECT
     first_seen_date AS `date`,
     CAST(REGEXP_EXTRACT(adjust_campaign, r' \((\d+)\)$') AS INT64) AS campaign_id,
     CAST(REGEXP_EXTRACT(adjust_ad_group, r' \((\d+)\)$') AS INT64) AS ad_group_id,
     SUM(repeat_user) AS repeat_users,
-    SUM(retained_week_4) AS retained_week_4
+    SUM(retained_week_4) AS retained_week_4,
   FROM
+    -- TODO: we should update this to use retention_clients view instead
```

Comment:
If I recall correctly, we want to deprecate this table in favour of retention_v1 and retention_clients view. @kwindau Do you have any objections against this? Also, if we do this change it should be done as a follow up PR.

Reply:
I think for the sake of just getting up to par with what they currently have in time for 12/15, it's probably ok to leave for now, and we can always revisit later